### DISCUSSION TITLE
Is it safe to distribute the anon key and URL within a demo application?

### DISCUSSION THREAD

**Discussion by maxcell** [supabase/supabase#7915] [Questions] [answered]
I couldn't quite figure out the answer for this but just wanted to check in on it. We have a demo application we want to show off with Supabase that includes Supabase Auth and a simple database. The app is just a simple CRD (create-read-delete) note taking application. But we have a lot of users who don't go through and setup their own database. We wanted to include the environment keys but we've also turned on RLS so that only a user could read, update, and delete their own notes and they can only read their own profile (the public schema we expose).

**Comment by silentworks** [comment, marked as answer]
Yes this is safe as long a RLS is turned on and you aren't sharing the service role key anywhere in the demo app distribution.
